client/my-sites/site-settings/jetpack-credentials/index.jsx

```diff
@@ -26,7 +26,8 @@ import {
 } from 'state/selectors';
 import {
 	updateCredentials as updateCredentialsAction,
-	autoConfigCredentials as autoConfigCredentialsAction
+	autoConfigCredentials as autoConfigCredentialsAction,
+	deleteCredentials as deleteCredentialsAction
```

Comment:
This renaming was discouraged by @dmsnell and I agree. It's not too useful and only pollutes things.